compiler/luci/import/src/ImporterEx.cpp

```diff
@@ -43,11 +51,14 @@ std::unique_ptr<Module> ImporterEx::importVerifyModule(const std::string &input_
   auto data_data = reinterpret_cast<uint8_t *>(model_data.data());
   auto data_size = model_data.size();
 
-  flatbuffers::Verifier verifier{data_data, data_size};
-  if (!circle::VerifyModelBuffer(verifier))
+  if (data_size < FLATBUFFERS_SIZE_MAX)
   {
-    std::cerr << "ERROR: Invalid input file '" << input_path << "'" << std::endl;
-    return nullptr;
+    flatbuffers::Verifier verifier{data_data, data_size};
+    if (!circle::VerifyModelBuffer(verifier))
+    {
+      std::cerr << "ERROR: Invalid input file '" << input_path << "'" << std::endl;
+      return nullptr;
+    }
   }
```

Comment:
If the size exceeds 2GB, VerifyModelBuffer will probably not be executed. Is that okay?

Reply:
Yes, not executed.
I can't say it's OK but Tensorflow is doing like this.